doc/api.rst

```diff
@@ -0,0 +1,478 @@
+        NameCollection(["label1", "label2"])
+    # Alternately, a list will be converted to a NameCollection
+    input_master.name_collection["labels"] = ["label1", "label2"]
+
+
+.. py:class:: Mean
+
+    A Numpy ``ndarray`` representing a distribution's mean, with a specialized 
+    string method for writing to the parameter file. Supports monovariate 
+    and multivariate distributions.
+
+    .. py:method:: __init__(self, *args)
+
+        Create a :py:class:`Mean` object storing the provided mean value or 
+        vector.
+
+        :param args: The mean value which is interpreted by the Numpy ``array`` constructor. 
+        :type args: array_like
+
+Usage example:
+
+.. code-block:: python
+
+    # Create a Mean object in the InputMaster mean InlineSection
+    input_master.mean["dist1"] = Mean(0.0)
+    # Alternately, a numeric value will be converted to a Mean
+    input_master.mean["dist2"] = 0.0
+
+
+.. py:class:: Covar
+
+    A Numpy ``ndarray`` representing a distribution's covariance, with a 
+    specialized string method for writing to the parameter file. Supports 
+    monovariate and multivariate distributions.
+
+    .. py:method:: __init__(self, *args)
+
+        Create a :py:class:`Covar` object storing the provided covariance 
+        value or vector.
+
+        :param args: The covariance value which is interpreted by the Numpy ``array`` constructor. 
+        :type args: array_like
+
+Usage example:
+
+.. code-block:: python
+
+    # Create a Covar object in the InputMaster covar InlineSection
+    input_master.covar["dist1"] = Covar(1.0)
+    # Alternately, a numeric value will be converted to a Covar
+    input_master.covar["dist2"] = 1.0
+
+
+.. py:class:: DPMF
+
+    A Numpy ``ndarray`` representing a DPMF with a specialized string method 
+    for writing to the parameter file. As it is intended for use in Gaussian 
+    mixture models, it supports monovariate distributions only. 
+
+    .. py:method:: __init__(self, *args)
+
+        Create a :py:class:`DPMF` object storing the provided distribution.
+
+        :param args: The probability distribution as an array of probabilties which is interpreted by the Numpy ``array`` constructor. 
+        :type args: array_like or multiple arguments
+
+    .. py:classmethod:: uniform_from_shape(self, shape)
+
+        A class method for creating a uniform DPMF with the specified shape.
+
+        :param shape: The shape of the DPMF, as its integer length.
+        :type shape: int
+        :returns: DPMF with given shape and uniform probabilities.
+        :rtype: DPMF
+
+Usage example:
+
+.. code-block:: python
+
+    # Create a custom DPMF object in the InputMaster mean InlineSection
+    input_master.dpmf["biased"] = DPMF([0.7, 0.3])
+    # Create a uniform DPMF with a specified shape
+    input_master.dpmf["uniform"] = DPMF.uniform_from_shape(3)
+
+
+.. py:class:: DiagGaussianMC
+
+    A Gaussian distribution with diagonal covariance. Currently the only 
```

Comment:
Is this correct? I thought it was referring to the diagonal components in a matrix. Also please define and expand any acronyms.

Reply:
Clarified this refers to a diagonal covariance matrix.